### PR TITLE
Domain only: Add domain delay notice to default screen

### PR DESCRIPTION
**Pull request by danhauk**
This addresses part of #11441 by adding the new domain delay notice to the default screen in place of the yellow warning notice. This creates a more positive experience for users who just purchased a domain because we don't immediately show them a big yellow warning.

image

Todo:

- [x] Only show this notice for the first six hours

#### Testing instructions

1. Start the server and visit http://calypso.localhost:3000/start/domain-first?new=newdomain.live where `newdomain.live` is a unique domain.
2. Continue through the checkout flow selecting "Just buy the domain" option.
3. Assert you see the notice about new domains on the default screen.
4. If you have one, switch to a domain-only site that was registered more than 48 hours ago.
5. Assert you do not see the notice about new domains on the default screen.

### CODE REVIEW

**Comment by gziolo** [comment]
@danhauk We don't return exact hour of domain registration from the API:
```registration_date: "March 13, 2017"```

What do you think about having this notice displayed from 24 up to 48 hours?

**Comment by gziolo** [comment]
I added commit with implementation that displays notice only when domain was purchased the same day or yesterday. Let me know if that works for you.

File: client/my-sites/upgrades/domain-management/list/domain-only.jsx [hunk 1]
```diff
@@ -10,19 +10,27 @@ import React, { PropTypes } from 'react';
...
+			title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
+			line={ translate( 'Start a site now to unlock everything WordPress.com can offer. Or add email and manage your domain preferences.' ) }
+			action={ translate( 'Create Site' ) }
+			actionURL={ `/start/site-selected/?siteSlug=${ encodeURIComponent( domainName ) }&siteId=${ encodeURIComponent( siteId ) }` }
+			secondaryAction={ translate( 'Manage Domain' ) }
+			secondaryActionURL={ domainManagementEdit( domainName, domainName ) }
+			illustration={ '/calypso/images/drake/drake-browser.svg' } />
+		{ hasNotice && (
+			<div className="domain-only-site__settings-notice">
+				{ translate( 'New domains usually start working immediately, but may be unreliable for the first few hours.' ) }
```
Comment (a8ci18n) [inline client/my-sites/upgrades/domain-management/list/domain-only.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Your domain should start working immediately, but may be unreliable during the first 72 hours.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by danhauk** [comment]
@gziolo I think 24-48 hours should work fine. cc @ranh in case there's any objections.

**Comment by danhauk** [comment]
Product works as expected. 👍

File: client/my-sites/upgrades/domain-management/list/domain-only.jsx (hunk 1 above)
Comment (gziolo) [inline client/my-sites/upgrades/domain-management/list/domain-only.jsx]: Let's update it before merge then :)

**Comment by drewblaisdell** [comment]
Code :+1: I updated the commit history to use the notice text from [this comment](https://github.com/Automattic/wp-calypso/pull/11916#discussion_r105668637).

**Comment by gziolo** [comment]
Thanks @drewblaisdell! :)

**Pull request merged**